### PR TITLE
Fsi updates 010621a

### PR DESCRIPTION
**Pull request by jbranson000**
#1017

### CODE REVIEW

File: components/mixins/experience-event/experienceevent-card-actions.example.1.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
```
Comment (mjasrotia) [inline components/mixins/experience-event/experienceevent-card-actions.example.1.json:1]: The example is missing the top level ""xdm:personalFinances" object

File: components/mixins/experience-event/experienceevent-card-application-process.example.1.json [hunk 2]
```diff
@@ -1 +1,26 @@
-{}
+{
```
Comment (mjasrotia) [inline components/mixins/experience-event/experienceevent-card-application-process.example.1.json:1]: The example is missing the top level ""xdm:personalFinances" object

File: components/mixins/experience-event/experienceevent-claim-process.example.1.json [hunk 3]
```diff
@@ -1 +1,31 @@
-{}
+{
```
Comment (mjasrotia) [inline components/mixins/experience-event/experienceevent-claim-process.example.1.json:1]: The example is missing the top level ""xdm:disputes" object

File: components/mixins/experience-event/experienceevent-contact-request-details.example.1.json [hunk 4]
```diff
@@ -1 +1,24 @@
-{}
+{
```
Comment (mjasrotia) [inline components/mixins/experience-event/experienceevent-contact-request-details.example.1.json:1]: The example is missing the top level ""xdm:customerFeedback" object

File: components/mixins/experience-event/experienceevent-credit-limit-increase-details.example.1.json [hunk 5]
```diff
@@ -1 +1,15 @@
-{}
+{
```
Comment (mjasrotia) [inline components/mixins/experience-event/experienceevent-credit-limit-increase-details.example.1.json:1]: The example is missing the top level ""xdm:personalFinances" object

**Review by mjasrotia — APPROVED** [review]

**Review by prabhum2 — APPROVED** [review]

**Review by prabhum2 — APPROVED** [review]

**Pull request merged**